solidity/dashboard/src/components/copy-stake-steps/Step2.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import Button from "../Button"
+import Badge from "../Badge"
+
+const options = [
+  {
+    id: 1,
+    type: "COPY_STAKE_FLOW",
+    title: "Copy stake balance to an upgraded delegation.",
+    subtitle:
+      "Avoid waiting the undelegation period and copy over your stake balance to a newly upgraded delegation. Your current stake will undelegate.",
+    recomended: true,
+  },
+  {
+    id: 2,
+    type: "WAIT_FLOW",
+    title:
+      "Undelegate and wait the undelegation period to stake to a newly upgraded delegation.",
+    subtitle:
+      "Undelegate and wait the 60 day undelegation period in order to stake on the upgraded staking contract. You can start new delegations with any tokens not already staked.",
```

Comment:
Does it make sense to parameterised "60 days" and not having it hard-written?

Reply:
We can have it hand-written, it's not something that's changing and we don't expect this flow to be something generic to use.